tests/sharepoint/nsb2019/test_mapping.py

```diff
@@ -69,6 +69,7 @@ def test_parser(self):
                 json.loads(p.json()) for p in mapped_procedures
             ]
             mapped_procedure_json.sort(key=lambda x: str(x))
+            print(mapped_procedure_json)
```

Comment:
We probably don't need this print statement. Also, does the version of aind-data-schema need to be incremented?

Reply:
We should probably increment it after the pydantic 2 upgrade.